Source/Core/VideoBackends/D3D/D3DPerfQuery.cpp

```diff
@@ -71,8 +71,8 @@ void PerfQuery::DisableQuery(PerfQueryGroup group)
 void PerfQuery::ResetQuery()
 {
   m_query_count.store(0, std::memory_order_relaxed);
-  for (size_t i = 0; i < m_results.size(); ++i)
-    m_results[i].store(0, std::memory_order_relaxed);
+  for (auto & m_result : m_results)
+    m_result.store(0, std::memory_order_relaxed);
```

Comment:
The `m_` prefix is often used to distinguish members from locals. Please don't use it for local variables. Just `result` is fine (same goes for the other two).
```suggestion
  for (auto& result : m_results)
    result.store(0, std::memory_order_relaxed);
```